{{cookiecutter.project_name}}/pyproject.toml

```
@@ -64,6 +64,9 @@ xfail_strict = true
 addopts = [
     "--import-mode=importlib",  # allow using test files with same name
 ]
+pythonpath = [
+    ".", "src"
```

The whole reason for the `src` layout is so that `.` doesn’t have to be in PYTHONPATH under any circumstances: https://docs.pytest.org/en/latest/explanation/goodpractices.html

```suggestion
    "src"
```

---

yes, but do we need  `pythonpath` at all, if we say everyone should install the package? (See discussion #119).

---

I agree, feel free to close it :)